src/Genre.elm

```diff
@@ -18,16 +21,6 @@ toCapital str =
     String.toUpper (String.left 1 str) ++ String.dropLeft 1 str
 
 
-{-| Initializes a multiselect with an available and selected list
--}
-genresMultiselectModel : String -> Set Genre -> Set Genre -> Multiselect.Model
-genresMultiselectModel elemId available selected =
```

Comment:
Moved and renamed.  Recommended practice in Elm is to not repeat the module name in function names if possible, and then to use the qualified version if there's ambiguity. e.g. `Genre.multiselectModel` instead of `genreMultiselectModel`